src/array_api_stubs/_draft/array_object.py

```diff
@@ -145,8 +155,9 @@ def __abs__(self: array, /) -> array:
         .. versionchanged:: 2022.12
             Added complex data type support.
         """
+        ...
 
-    def __add__(self: array, other: Union[int, float, array], /) -> array:
+    def __add__(self: array, other: int | float | array, /) -> array:
```

Comment:
Just checking: if `other` is an array, is it always intended for it to be of the same type as `self`? No `other` subclasses? No different `self` subclass? If no, then it'd be best to annotate it without use of the type variable:
```suggestion
    def __add__(self: array, other: int | float | Array, /) -> array:
```

Reply:
Agreed 👍

Reply:
Done.